api/views.py

```diff
@@ -383,7 +383,8 @@ def handle_post(self, request, *args, **kwargs):
             return bad_request('Must include an `email` property')
 
         try:
-            user = User.objects.get(email=body['email'])
+            req_email = body['email'].lower()
+            user = User.objects.annotate(email_lower=Lower('email')).get(email_lower=req_email)
```

Comment:
@GregoryHorvath You can use [`iexact`](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#std:fieldlookup-iexact) here directly.

```suggestion
            user = User.objects.get(email__iexact=body['email'])
```